tfs/src/main/java/hudson/plugins/tfs/listeners/JenkinsRunListener.java

```diff
@@ -54,4 +88,46 @@ private String getStartedBy(final Run run) {
         }
         return startedBy;
     }
+
+    private TeamGitStatus trySetPullRequestStatus(final Run run, @Nonnull final TaskListener listener, final GitStatusState buildState, final String buildDescription, final String targetUrl) {
+        try {
+            final TeamGitStatus status = new TeamGitStatus();
+            status.state = buildState;
+            status.description = buildDescription;
+            status.targetUrl = targetUrl;
+            status.context = new GitStatusContext("ci-build", "jenkins-plugin");
+
+            if (!UnsupportedIntegrationAction.isSupported(run, listener)) {
+                final PrintStream logger = listener.getLogger();
+                logger.print("NOTICE: ");
```

Comment:
Can you combine these calls for readability?

Reply:
Sure, refactored it out and remove unnecessary logging for readability.